.rubocop.yml

```diff
@@ -113,6 +113,11 @@ Lint/TopLevelReturnWithArgument:
 Lint/TrailingCommaInAttributeDeclaration:
   Enabled: true
 
+Layout/TrailingWhitespace:
+  AllowInHeredoc: true
+  Exclude:
+    - lib/bolt/bolt_option_parser.rb
```

Comment:
```suggestion
```

This isn't needed with `AllowInHeredoc: true`.
We should also update the Gemfile to use rubocop ~> 1.0.